public/apps/configuration/panels/role-edit/cluster-permission-panel.tsx

```diff
@@ -44,7 +44,12 @@ export function ClusterPermissionPanel(props: {
         >
           <EuiFlexGroup>
             <EuiFlexItem className={LIMIT_WIDTH_INPUT_CLASS}>
-              <EuiComboBox options={optionUniverse} selectedOptions={state} onChange={setState} />
+              <EuiComboBox
+                placeholder="Search for action group name or permission name"
```

Comment:
Did you verify if it works on UI. In my memory setting a place holder for a combobox doesn't work for me.

Reply:
Yes it works on UI. 
![image](https://user-images.githubusercontent.com/63824432/91231343-edcebb00-e6e1-11ea-8f06-8f6279754b79.png)